src/server/action/approve_business.tsx

```diff
@@ -0,0 +1,19 @@
+"use server";
+
+import { db } from "~/server/db";
+import { business } from "~/server/db/schema";
+import { eq } from "drizzle-orm";
+
+export async function approveBusiness(businessID: number) {
+  try {
+    await db
```

Comment:
update query logic should not be on server action file it should be on query file on the "server-only" file